pygeos/set_operations.py

```diff
@@ -193,3 +196,93 @@ def union_all(geometries, axis=0, **kwargs):
     # create_collection acts on the inner axis
     collections = lib.create_collection(geometries, GeometryType.GEOMETRYCOLLECTION)
     return lib.unary_union(collections, **kwargs)
+
+
+@requires_geos("3.8.0")
+def coverage_union(a, b, **kwargs):
+    """Merges multiple polygons into one. This is an optimized version of
+    union which assumes the polygons to be non-overlapping.
+
+    Requires at least GEOS 3.8.0.
+
+    Parameters
+    ----------
+    a : Geometry or array_like
+    b : Geometry or array_like
+
+    See also
+    --------
+    coverage_union_all
+
+    Examples
+    --------
+    >>> polygon = Geometry("POLYGON ((0 0, 0 1, 1 1, 1 0, 0 0))")
+    >>> actual = coverage_union(polygon, Geometry("POLYGON ((1 0, 1 1, 2 1, 2 0, 1 0))"))
+    >>> from .predicates import equals
+    >>> expected = Geometry("POLYGON ((0 0, 0 1, 1 1, 2 1, 2 0, 1 0, 0 0))")
```

Comment:
I wanted to suggest avoiding `equals` to simplify the doctests, but what I'm seeing indicates that the vertices in the polygon are reordered by the operation.  E.g., 

```
>>> coverage_union(polygon, Geometry("POLYGON ((1 0, 1 1, 2 1, 2 0, 1 0))"))
<pygeos.Geometry POLYGON ((2 1, 2 0, 1 0, 0 0, 0 1, 1 1, 2 1))>

Union with None returns same polygon
>>> coverage_union(polygon, None)
<pygeos.Geometry POLYGON ((1 1, 1 0, 0 0, 0 1, 1 1))>
```

Is that why you went with `equals` here?

Reply:
Nevermind, just saw 5cb06b0b3cdfd70ba7cb70e267b3558616c29694

However, it would be good to use fully qualified import for `pygeos` in docstrings: `from pygeos.predicates import equals`

Reply:
Yes, that is the reason I had to use `equals`.

I have now changed it to use absolute import instead of relative import.